api/store/store.go

```diff
@@ -0,0 +1,51 @@
+package store
+
+import (
+	"net/http"
+	"time"
+
+	"github.com/screwdriver-cd/sd-cmd/api/screwdriver"
+)
+
+const (
+	timeoutSec = 10
```

Comment:
This means every sd-cmd fails with timeout by 10 sec? It might better to change it adjustable by each commands.

Reply:
It would be difficult to change the time by each commands. 

I thought client timeout include just connection time, so 10 sec is enough. 
However when I read [client.go source](https://golang.org/src/net/http/client.go), that say the timeout include the connection time, any redirects, and reading the response body.

10 second is definitely not enough to download big binary commands. 

How about to 180 second?

Reply:
I thought client timeout include just connection time, so 10 sec is enough. 
However when I read [client.go source](https://golang.org/src/net/http/client.go), that say the timeout include the connection time, any redirects, and reading the response body.

As you said 10 second is definitely not enough to download big binary commands. 

How about 180 second?

Reply:
Understood. Constant value is ok if enough value is set.